src/index.ts

```diff
@@ -177,31 +196,44 @@ export function createIframe({
   loadingTimeoutInMs,
   themeOptions,
   analyticsInfo,
-  classSlug,
+  pageId,
 }: CreateIframeOptions): IframeInstance {
   if (targetPage == 'joinClass' && !sessionId) {
     throw new Error('sessionId is required if you are loading a join class page.')
   }
 
-  if (targetPage == 'class' && !classSlug) {
-    throw new Error('classSlug is required if you are loading a class page.')
```

Comment:
We are no longer checking for **classSlug** (or **classId**) and it could cause downstream errors now.

Reply:
The `class` page was only a thing for a version of the embed that we don't use any more (at least not with this package).

The `classSlug` was required for the `class` page as the slug was required in the url for that page. But for all the other pages we have the slug only used but the page hosting the iframe, we don't use it inside the iframe anywhere. We just pass it to the hosting page so they can use it for SEO if they want.

But you are correct about checking `classId`, that is required if the page is the `watch` page. Will add check for that.